Win32Emu/Win32/Modules/MsvcrtModule.cs

```diff
@@ -41,6 +41,14 @@ public class MsvcrtModule : IWin32ModuleUnsafe
 		private const byte SHIFTJIS_LEAD_BYTE_RANGE2_START = 0xE0;
 		private const byte SHIFTJIS_LEAD_BYTE_RANGE2_END = 0xFC;
 		
+		// Random number generator state (thread-specific in real MSVCRT, but we use a simple global)
+		private uint _randomSeed = 1;
```

Comment:
Can we set this to a proper random seed? Generated on construction or initialisation?